src/app/component/general/eco-news/eco-news-detail/eco-news-detail.component.html

```diff
@@ -1 +1,63 @@
-<p>eco-news-detail works!</p>
+<div class="container">
+  <div class="top-elements">
+    <div class="back-button">
+      <a [routerLink]="['/news']"
+         class="button-link">
+        <div class="button-content">
+          <div class="button-arrow">
+            <img src="../../../../assets/img/icon/econews/arrow_left.svg"
```

Comment:
Move pictures in variable in extra file and use binding

Reply:
not required